Gemfile.lock

```diff
@@ -368,5 +377,8 @@ DEPENDENCIES
   webmock (~> 1.2)
   whenever
 
+RUBY VERSION
```

Comment:
Can you re-bundle this with your dev VM please? Otherwise people will get warnings on their own VMs until the ruby version this app uses is bumped.